pyderman/drivers/chrome.py

```diff
@@ -25,13 +26,13 @@ def get_url(
         resolved_version = version
     if not resolved_version:
         raise ValueError(f"Unable to locate ChromeDriver version! [{version}]")
-    if _os == "mac-m1":
+    if _os == MAC_ARM:
         _os = "mac"
```

Comment:
You'll see this pattern in a few places; where I use the const for the os in one place but not the following line.  This is to differentiate between the value needed in url vs what pyderman uses to identify the os.